sample/src/androidTest/java/com/schibsted/spain/barista/sample/TextInputLayoutWithTextTest.kt

```diff
@@ -0,0 +1,39 @@
+package com.schibsted.spain.barista.sample
+
+import androidx.test.rule.ActivityTestRule
+import com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertDisplayed
+import com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertNotDisplayed
+import com.schibsted.spain.barista.interaction.BaristaEditTextInteractions.writeTo
+import com.schibsted.spain.barista.internal.failurehandler.BaristaException
+import org.junit.Rule
+import org.junit.Test
+
+class TextInputLayoutWithTextTest {
+
+  @get:Rule
+  var activityRule = ActivityTestRule(HintAndErrorActivity::class.java)
+
+  @Test
+  fun assertText() {
+    writeTo(R.id.hintanderror_inputlayout_editText, "Test text")
+    assertDisplayed(R.id.hintanderror_inputlayout, "Test text")
+  }
+
+  @Test
+  fun assertNoText() {
+    writeTo(R.id.hintanderror_inputlayout_editText, "Test text")
+    assertNotDisplayed(R.id.hintanderror_inputlayout, "Other test text")
+  }
+
+  @Test(expected = BaristaException::class)
```

Comment:
You could test the exception more explicitly, to check that the error message is right.
Like this one from ListClickTest:
```

  @Test
  public void fail_whenNoViewFound() {
    launchTestActivity(ListsActivity.buildIntent());

    Throwable thrown = catchThrowable(() -> clickListItem(20));

    spyFailureHandlerRule.assertEspressoFailures(1);
    assertThat(thrown).isInstanceOf(BaristaException.class)
        .hasMessage("No ListView or RecyclerView found in the hierarchy");
  }
 ```

Reply:
Uhmm, it's a generic error message.. not sure if I will add a custom message here